internal/status/svc_annotation.go

```diff
@@ -54,7 +54,7 @@ func CheckUpdateSvcAnnotation(key, namespace, name string) bool {
 		utils.AviLog.Errorf("key: %s, msg: there was an error in updating the Service annotation for NPL: %v", key, err)
 		return false
 	}
-	utils.AviLog.Debugf("key: %s, msg: updated NPL annotation from Service: %s/%s", key, namespace, name)
+	utils.AviLog.Infof("key: %s, msg: updated NPL annotation from Service: %s/%s", key, namespace, name)
```

Comment:
Do we really need an Infof for this log? Its completely your call though.

Reply:
I would prefer to keep this, else we would have no idea about the status. I think we do this for some other status updates too.